packages/quick_actions/quick_actions/pubspec.yaml

```diff
@@ -17,15 +17,18 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0
+  quick_actions_platform_interface: ^1.0.0
 
 dev_dependencies:
-  test: ^1.16.3
   flutter_test:
     sdk: flutter
   integration_test:
     sdk: flutter
-  pedantic: ^1.10.0
+  mockito: ^5.0.0-nullsafety.7
```

Comment:
I think we can use the stable `5.0.2` version of mockito here.

Reply:
Bumping this to 5.0.2 result in failing version solving:
```
Because mockito >=5.0.0 depends on analyzer ^1.0.0 which depends on crypto ^3.0.0, mockito >=5.0.0 requires crypto ^3.0.0.
And because every version of integration_test from sdk depends on crypto 2.1.5, mockito >=5.0.0 is incompatible with integration_test from sdk.
```